### PR TITLE
🔨 chore: add script to run all test case

### PR DESCRIPTION
**Pull request by jin860315**
----

*Checklist:*

- README.md includes:
  - [ ] The details of signing data and transaction data, this includes parameters.
  - [ ] Official docs or white paper.
  - [ ] Website or api can query assets and broadcast transaction.

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

 
 **PR Summary by Typo**
------------

 **Summary**

Introduced new script "scripts/run-all-test.sh" for running tests in multiple scopes using Lerna. Set up test environments and executed bootstrap, build, and test commands.

**Key Points**

* Introduced new script for running tests across multiple scopes
* Utilized Lerna for test setup and execution
* Executed bootstrap, build, and test commands for each scope 

 <h6>To turn off PR summary, please visit <a href="https://app.typoapp.io/settings/dev-analytics/notification?tab=codeHealth">Notification settings</a>.</h6>

### CODE REVIEW

File: scripts/run-all-test.sh [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  --scope @coolwallet/cosmos
+  --scope @coolwallet/xtz
+  --scope @coolwallet/atom 
+)
+LERNA="npx lerna"
+
+$LERNA bootstrap $TEST_FIXTURE_SCOPES
+$LERNA run build $TEST_FIXTURE_SCOPES
+$LERNA bootstrap "${TEST_SCOPES[@]}"
+$LERNA run build "${TEST_SCOPES[@]}"
```
Comment (tp26610) [inline scripts/run-all-test.sh:23]: 跑測試的 coin module 不需要 build

Comment (jin860315) [reply scripts/run-all-test.sh]: 這邊～
8b44a820c7e3f27707677aceaf985125f101ba24

Comment (tp26610) [reply scripts/run-all-test.sh]: Test fixture 需要 build，只有 coin module 不用

Comment (jin860315) [reply scripts/run-all-test.sh]: 改改
3941fb74a2199c411e7e88683ca3bb5337c70f4b

**Review by tp26610 — APPROVED** [review]

**Review by gundomlegend — APPROVED** [review]

**Pull request merged**